src/ffi/mod.rs

```diff
@@ -80,6 +91,8 @@ pub fn set_global_logger(log_level: TelioLogLevel, logger: Box<dyn TelioLoggerCb
     let tracing_subscriber = TelioTracingSubscriber {
         callback: logger,
         max_level: log_level.into(),
+        #[cfg(feature = "hide_ips")]
+        censor: LogCensor::new(),
```

Comment:
To make it more nicely composable, recomended way would be to make cencor a layer you optionaly add https://docs.rs/tracing-subscriber/latest/tracing_subscriber/layer/index.html.

Reply:
I saw it, but don't really see the benefits - what are the gains from that?

Reply:
So primary benefit that it would make it more composable, but i guess we can leave this for then we want to reuse this at some point.

Reply:
For examples if we would want to test it out with tcli, but leaving for you to decide, if you considered it and did not find it usefull, leaving it to your decision.